app/assets/javascripts/school-search/data.js

```diff
@@ -0,0 +1,97 @@
+export const allSchools = [
+  {
+    urn: 123456,
+    name: 'Corfe Hills School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: false,
+    rural: false,
+    alreadyPartnered: false,
+    partnershipConfirmed: false
+  },
+  {
+    urn: 123456,
+    name: 'Rural school',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: false,
+    rural: true,
+    alreadyPartnered: false,
+    partnershipConfirmed: false
+  },
+  {
+    urn: 123456,
+    name: 'Pupil Premium School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: true,
+    rural: false,
+    alreadyPartnered: false,
+    partnershipConfirmed: false
+  },
+  {
+    urn: 123456,
+    name: 'Rural PP School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: true,
+    rural: true,
+    alreadyPartnered: false,
+    partnershipConfirmed: true
+  },
+  {
+    urn: 123456,
+    name: 'Corfe Hills School',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: false,
+    rural: false,
+    alreadyPartnered: true,
+    partnershipConfirmed: false
+  },
+  {
+    urn: 123456,
+    name: 'School With A Very Very Very Very Very Very Very Long Name',
+    address: 'Higher Blandford Road, Broadstone, Dorset, BH18 9BG',
+    phase: 'secondary',
+    phaseDisplay: 'Secondary',
+    type: 'academy',
+    typeDisplay: 'Academy',
+    pupilPremium: false,
+    rural: false,
+    alreadyPartnered: false,
+    partnershipConfirmed: true
+  }
+]
+
+export const allMats = [
```

Comment:
What does MAT stand for in this context?

Reply:
multi academy trust